cross/tvheadend/Makefile

```diff
@@ -1,7 +1,7 @@
 PKG_NAME = tvheadend
-PKG_VERS = 3.4patch1
+PKG_VERS = 4.0.9
 PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
```

Comment:
More or less. You're right: `PKG_DIST_NAME` is usually `$(PKG_NAME)-$(PKG_VERS)`, but upstream doesn't always use that format for various reasons.
For spksrc, that format ensures the filename is uniquely connected to a cross package: no possibility for duplicates in DISTRIB_DIR for example, and it's easier to recognize too.

See [here](https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.cross-cc.mk#L9). Essentially, we're overriding `PKG_DIST_NAME` (the original/upstream filename) with `PKG_DIST_FILE` wherever it's used.